app/src/main/java/com/cmput301f20t21/bookfriends/ui/profile/ProfileFragment.java

```diff
@@ -47,6 +63,45 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
         emailAddress = view.findViewById(R.id.email);
         phoneNumber = view.findViewById(R.id.phone);
         editProfile = view.findViewById(R.id.image_edit);
+        name = view.findViewById(R.id.username);
+
+
+        // get the login information from firebase
+        User firebaseUser = AuthService.getInstance().getCurrentUser();
+        if (firebaseUser != null) {
+            String userId = firebaseUser.getUid();
+            String username = firebaseUser.getUsername();
+            String email = firebaseUser.getEmail();
+            name.setText(username);
+            emailAddress.setText(email);
+
+            // search for field "phone"
+            final DocumentReference docReference = FirebaseFirestore.getInstance().collection("users").document(userId);
```

Comment:
this is directly connect to firebase as well and need to be in AuthService

Did we discuss that we might not need a phone number field? Let's bring it up in tonight's meeting and discuss